docs/AboutUs.adoc

```diff
@@ -4,53 +4,56 @@ ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-The dummy content given below serves as a placeholder to be used by future forks of the project. +
-{empty} +
-We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
+Augustine is a Contact Management software that was further developed from the original AddressBook-level 4 that was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
 
 == Project Team
 
-=== John Doe
+=== Damith
 image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+Role: Lecturer
 
-Role: Project Advisor
+'''
+
+=== Akshay Narayan
+image::akshay.jpg[width="150", align="left"]
+{empty}[https://github.com/okkhoy[github]]
+
+Role: Tutor
 
 '''
 
-=== John Roe
-image::lejolly.jpg[width="150", align="left"]
-{empty}[http://github.com/lejolly[github]] [<<johndoe#, portfolio>>]
+=== Ritchie Lam
+image::ritchie.jpg[width="150", align="left"]
+{empty}[http://github.com/ritchielq[github]] [<<ritchie#, portfolio>>]
 
-Role: Team Lead +
-Responsibilities: UI
+Role: Developer +
+Responsibilities: Edit
```

Comment:
we would like to see your responsibilities in terms of the component you are in charge of.